server/tabby_server/__main__.py

```diff
@@ -1,9 +1,39 @@
 from flask import Flask, request
 from http import HTTPStatus
+# from services import resource_format  # Use when actually running the server
+from .services import resource_format  # Use when running pytest
+# import test_routes
+
+# --- IF USING PYTEST VS PYTHON ---
+# You only need to modify this file! I've changed it around so only this
+# breaks pytest or python! No need to modfy additional files beyond!
 
 app = Flask(__name__)
 
 
+# @app.route('/testpage', methods=['GET'])
+# app.add_url_rule('/testpage', view_func=alternate_page.test_page)
+# @app.route('/test/make_request', methods=['GET'])
```

Comment:
This API route appears to be lazy loading where it would be difficult to specify what kind of HTTP Route we want. Can we look into a better way of implementing the separate files. For example,
We can leave the app.routes same as how we had it before and if we want to have the code in a separate file, then we can have it call a function that is external to the main. Then main would just import that external file for the function and all that.